node/src/blockchain/errors/validation_status.rs

```diff
@@ -261,7 +246,7 @@ mod tests {
     // #[test]
     // fn previous_attempts_hash_works_correctly() {
     //     let now = SystemTime::now();
-    //     let clock = ValidationFailureClockMock::default()
+    //     let clock = SimpleClockMock::default()
```

Comment:
Why are we keeping this test like this, as a comment, shouldn't we decide whether we want to keep it or not?

Reply:
I was on the track to prove a hypothesis that this code structure, PreviousAttempts, doesn't need the Hash impl. I disabled it together with this test that used to cover it. It turned out right.I just forgot to delete this test.